applications/nublado/values-ccin2p3.yaml

```diff
@@ -64,11 +69,22 @@ controller:
         - containerPath: "/sps"
           volumeName: "sps"
 jupyterhub:
+  prePuller:
+    continuous:
+      enabled: true
   hub:
+    baseUrl: "/nb"
+    config:
+      ServerApp:
+        shutdown_no_activity_timeout: 432000
```

Comment:
This doesn't do anything any more, which is why we deleted it.

Reply:
OK, I removed useless lines.